ansible/roles/texlive/templates/texlive-path.sh.j2

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+export PATH=$PATH:{{ texlive_TEXDIR }}/bin/x86_64-linux/
```

Comment:
$PATH should come last--we're overriding it, so having it come first could hide stuff we need to take priority.